packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -633,12 +634,20 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 			}
 		});
 
-		await super.initialize(connectMessage, timeout).finally(() => {
+		try {
+			await super.initialize(connectMessage, timeout);
+			if (this._details) {
+				if (!this.details.supportedFeatures) {
+					this._details.supportedFeatures = {};
+				}
+				this._details.supportedFeatures![enable_single_commit_summary] = true;
```

Comment:
warning: nits ahead!

I thought we had a lint rule about `!`. Another way to do this:

`this._details.supportedFeatures = { ...this.details.supportedFeatures, enable_single_commit_summary: true }`

Not sure which is better, mine clones the thing every time, but it's only one line :)